docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,274 @@
 Latent Tuberculosis
 ===================
 
-.. todo:
+Cause Model Diagram
+-------------------
+.. image:: joint_model_diagram.svg
 
-   all of this
+Data Description
+----------------
+.. list-table:: **Abbreviations**
```

Comment:
We came up with a standardized format for these tables (see measles, diarrheal diseases, neonatal enceph for examples). So we have a 'state' 'state name' and also 'definition' column for this particular table.

I think it would also make sense to have a row for every bubble in your diagram--so one for S_TB, I_HIV and another for S_TB, S_HIV, etc.